config/webpack.js

```diff
@@ -20,13 +20,15 @@ const pages = {
 
 const htmlHeadTemplate = _.template(fs.readFileSync('./html/includes/_head.ejs', 'utf8'));
 const htmlFooterTemplate = _.template(fs.readFileSync('./html/includes/_footer.ejs', 'utf8'));
+const htmlIconsTemplate = _.template(fs.readFileSync('./html/includes/_icons.ejs', 'utf8'));
```

Comment:
I think `.svg` is not going to work  (the file contains HTML hides SVG definitions). I just moved as `html` file to a `statics` folder, is it OK for you? We still import it as we did.